### PR TITLE
Adds new columns in template for showing each person's star number

### LINKED ISSUE #21: Add a column with the number of stars
The primary factor for ranking is the number of stars the user has.  
At the moment, it is not displayed. 
We may want to add a column displaying it left from the current score (which we may want to rename to make it clear it's just a tiebreaker)

### PR DESCRIPTION
**Pull request by Cpavrai**
resolves #21

### CODE REVIEW

File: src/main/resources/templates/index.ftl [hunk 1]
```diff
@@ -79,6 +79,9 @@
                                     </#if>
                                 </#list>
                             </td>
+                            <td>
+                                <span>${user.getStarNumber() / 2}</span>
```
Comment (olevitt) [inline src/main/resources/templates/index.ftl:83]: Are we sure this renders well when the user has an uneven number of stars ?

Comment (Cpavrai) [reply src/main/resources/templates/index.ftl:83]: If the question is : does it show 0 when people doesn't have done any challenge, the answer is yes, it shows 0.

**Review by olevitt — COMMENTED** [review]
Looks good to me

**Review by olevitt — APPROVED** [review]
:+1:

**Pull request merged**